kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -695,6 +732,7 @@ def _method_to_apr_proof(
         if method_name != 'setUp' and 'setUp' in contract.method_by_name:
             setup_digest = f'{contract_name}.setUp:{contract.digest}'
             _LOGGER.info(f'Using setUp method for test: {test}')
+            _LOGGER.info(f'method.digest: {method.digest}')
```

Comment:
I'm guessing this is also for debugging and can be removed?

Reply:
Removed